### DISCUSSION TITLE
Issue with forever Loading Times on Vercel-Hosted Websites

### DISCUSSION THREAD

**Discussion by zelal-dev** [vercel/community#7185] [Account Issue]
### Question

Hi team

I have hosted many websites on Vercel, but there is a problem: when I browse any website hosted on Vercel, it takes too much time to load. If I disconnect the internet and try again, it works fine. Alternatively, if I change my VPN, it also works fine.

This also happens when I browse the Vercel site itself.

I believe there might be an issue with Vercel and my IP or my internet provider.

I am not sure what the main reason is, which is why I am contacting you for support.

If it is not a Vercel issue, I will contact my internet provider.

Here is my IP: 103.23.254.4

Best WIshes
Zelal

**Comment by VidushaSanidu** [comment]
same issue for me. I can't access nextjs, vercel, nextUI sites from one of my ip address.

[a comment by an automated account is not shown]
